.github/workflows/build.yml

```diff
@@ -39,3 +41,142 @@ jobs:
           path: codex-frontend.tar.gz
           # default:
           # retention-days: 90
+
+  deploy:
+    name: Docker image build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'schedule'
+        with:
+          ref: develop
```

Comment:
The schedule is meant to be a nightly snapshot of the current codebase.  On Houston, this is indeed the `main` branch because we merge the latest code into `main` and use tags to differentiate releases.  I think it is largely up to Ben for which branch he wants to target on the FE repo.  

Since the FE main branch is only updated with tagged releases, it will be frozen until a new release comes out (which is covered by the `latest` tag).  I'd argue `develop` since it is roughly the current stable state of the most bleeding-edge version.

Reply:
Yep I think we want to target develop for bleeding edge/nightly snapshot